src/components/Toast alerts/FailUserFetch.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Alert, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import { useTranslation } from 'react-i18next';
+
+export default function FailUserFetch() {
```

Comment:
please create a generic error/success toast notification. If we add a toast for each request in a project we would end up with ~100 toast components. The message  'fail_fetch_users' has to be received via component communication, also the color and the severity.

Reply:
created generic toast component